src/types/index.tsx

```diff
@@ -36,7 +37,7 @@ export type UserDetails = {
   cnic: string;
   ehad_duration: string;
   shift: string;
-  role: UserRole;
+  role: UserRoleType;
```

Comment:
This looks wrong. Why switch to type instead of enum?

Reply:
Sir , I didn't switch to type instead of enum , i just renamed type